src/framework/vst/internal/vstpluginrepository.cpp

```diff
@@ -0,0 +1,136 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2021 MuseScore BVBA and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program; if not, write to the Free Software
+//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+//=============================================================================
+
+#include "vstpluginrepository.h"
+
+#include "vstplugin.h"
+#include "log.h"
+
+using namespace mu;
+using namespace io;
+using namespace vst;
+
+//@see https://developer.steinberg.help/pages/viewpage.action?pageId=9798275
+static const std::string VST3_PACKAGE_EXTENSION = "*.vst3";
+
+Ret VstPluginRepository::loadAvailablePlugins()
+{
+    Ret result(true);
+
+    for (const std::string& pluginPath : pluginPathsFromDefaultLocation()) {

```

Comment:
better use `io::path`

Reply:
std::string is coming from Steinberg::Vst::PathList here, I'd leave it as it is since it's a private function